docs/community/release-notes.md

```diff
@@ -34,6 +34,91 @@ You can determine your currently installed version using `pip show`:
 
 ---
 
+## 3.15.x series
+
+### 3.15.0
+
+Date: 15th March 2024
+
+* Django 5.0 and Python 3.12 support [[#9157] (https://github.com/encode/django-rest-framework/pull/9157)]
+* Use POST method instead of GET to perform logout in browsable API [[9208] (https://github.com/encode/django-rest-framework/pull/9208)]
+* Added jQuery 3.7.1 support & dropped previous version [[#9094](https://github.com/encode/django-rest-framework/pull/9094)]
+* Use str as default path converter [[#9066](https://github.com/encode/django-rest-framework/pull/9066)]
+* Document support for http.HTTPMethod in the @action decorator added in Python 3.11 [[#9067](https://github.com/encode/django-rest-framework/pull/9067)]
+* Update exceptions.md [[#9071](https://github.com/encode/django-rest-framework/pull/9071)]
+* Partial serializer should not have required fields [[#7563](https://github.com/encode/django-rest-framework/pull/7563)]
+* Propagate 'default' from model field to serializer field. [[#9030](https://github.com/encode/django-rest-framework/pull/9030)]
+* Allow to override child.run_validation call in ListSerializer [[#8035](https://github.com/encode/django-rest-framework/pull/8035)]
+* Align SearchFilter behaviour to django.contrib.admin search [[#9017](https://github.com/encode/django-rest-framework/pull/9017)]
+* Class name added to unknown field error [[#9019](https://github.com/encode/django-rest-framework/pull/9019)]
+* Fix: Pagination response schemas. [[#9049](https://github.com/encode/django-rest-framework/pull/9049)]
+* Fix choices in ChoiceField to support IntEnum [[#8955](https://github.com/encode/django-rest-framework/pull/8955)]
+* Fix `SearchFilter` rendering search field with invalid value [[#9023](https://github.com/encode/django-rest-framework/pull/9023)]
+* Fix OpenAPI Schema yaml rendering for `timedelta` [[#9007](https://github.com/encode/django-rest-framework/pull/9007)]
+* Fix `NamespaceVersioning` ignoring `DEFAULT_VERSION` on non-None namespaces [[#7278](https://github.com/encode/django-rest-framework/pull/7278)]
+* Added Deprecation Warnings for CoreAPI [[#7519](https://github.com/encode/django-rest-framework/pull/7519)]
+* Removed usage of `field.choices` that triggered full table load [[#8950](https://github.com/encode/django-rest-framework/pull/8950)]
+* Permit mixed casing of string values for `BooleanField` validation [[#8970](https://github.com/encode/django-rest-framework/pull/8970)]
+* Fixes `BrowsableAPIRenderer` for usage with `ListSerializer`. [[#7530](https://github.com/encode/django-rest-framework/pull/7530)]
+* Change semantic of `OR` of two permission classes [[#7522](https://github.com/encode/django-rest-framework/pull/7522)]
+* Remove dependency on `pytz` [[#8984](https://github.com/encode/django-rest-framework/pull/8984)]
+* Fix validation for ListSerializer [[#8979](https://github.com/encode/django-rest-framework/pull/8979)]
```

Comment:
> #8979 is marked as "breaking change" and the fix for that (#9244) is not merged yet. It should be noted in the release notes this is a _breaking-change_.

@TGoddessana please see same implemented @, thank you,

Reply:
Although I'm not a maintainer, I'm very interested in the maintenance of the project, so I took a look at the issues.

1. one PR (#8979) that solves the original issue (#8926) seems to have been merged already, but the better implementation (#9244) is still not fully resolved based on the last comment in the issue.

2. so which decision do we make? 
- Should we revert #8926 if we need a better implementation? If a better design exists, would it be a good decision to include it in the 3.15 release?
- Do we need to rush #9244 to completion? Can we get it done before the release?
- Maybe we could revert #8926 and ship a bugfix release with 3.15.1 with a better implementation? It was clearly labeled as a breaking change, but it felt more like a bug fix to me.

Would love to hear your thoughts @tomchristie 🥲

Reply:
IMO the problem is that #8926 was addressed ignoring the advice regarding ["Customizing multiple update"](https://www.django-rest-framework.org/api-guide/serializers/#customizing-multiple-update)

> By default the `ListSerializer` class does not support multiple updates. This is because the behavior that should be expected for insertions and deletions is ambiguous.
>
> To support multiple updates you'll need to do so explicitly. When writing your multiple update code make sure to keep the following in mind:
>
>- How do you determine which instance should be updated for each item in the list of data?
>- How should insertions be handled? Are they invalid, or do they create new objects?
>- How should removals be handled? Do they imply object deletion, or removing a relationship? Should they be silently ignored, or are they invalid?
>**How should ordering be handled? Does changing the position of two items imply any state change or is it ignored?**
>
> **You will need to add an explicit id field to the instance serializer**. The default implicitly-generated id field is marked as read_only. This causes it to be removed on updates. Once you declare it explicitly, it will be available in the list serializer's update method.

Thus the proposed solution (#8979) only addesses a single use case while ignoring out many others which are now not supported by the framework. For example: using `ListSerializer` to perform in a bulk request a "create-update-delete" of a list of objects which was possible before this change.

Reply:
I was thinking about reverting that for this release

Reply:
If this change is not released in 3.15, which release do you think is the right one to fix and release it in: 3.16 or 3.15.1?

Reply:
if it is just reverted it will not be included in 3.15, as it is a big thing, it should be included in a major release like 3.16/3.17

Reply:
I just want to avoid including any not fully functional feature for 3.15

Reply:
> I just want to avoid including any not fully functional feature for 3.15

I agree with you. Thank you for your efforts!